guide/book.toml

```
@@ -0,0 +1,23 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[book]
+title        = "Google Cloud Client Libraries for Rust"
+authors      = ["Carlos O'Ryan"]
+language     = "en"
+multilingual = false
+src          = "src"
+
+[rust]
+edition = "2018"
```

I think everything else is 2021.
```suggestion
edition = "2021"
```

---

Done